inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplImportExportIntegrationTest.java

```diff
@@ -247,6 +249,7 @@ private KnowledgeBase buildKnowledgeBase(Project project, String name) {
         kb.setDescriptionIri(RDFS.COMMENT);
         kb.setLabelIri(RDFS.LABEL);
         kb.setPropertyTypeIri(RDF.PROPERTY);
+        kb.setExplicitlyDefinedRootConcepts(new ArrayList<IRI>());
```

Comment:
Is it necessary to define the generic type (IRI) here or can't you just use the diamond operator `<>`?

Reply:
changed it.